android/java/org/chromium/chrome/browser/crypto_wallet/fragments/PortfolioFragment.java

```diff
@@ -170,4 +226,116 @@ private void setUpCoinList(View view) {
     public void onAssetClick() {
         Utils.openAssetDetailsActivity(getActivity());
     }
+
+    // TODO(AlexeyBarabash): this is duplicated over other wallet UI classes
+    @Override
+    public void onConnectionError(MojoException e) {
+        mAssetRatioController = null;
+        InitAssetRatioController();
+    }
+
+    // TODO(AlexeyBarabash): this is duplicated over other wallet UI classes
+    public AssetRatioController getAssetRatioController() {
+        return mAssetRatioController;
+    }
+    private void InitAssetRatioController() {
+        if (mAssetRatioController != null) {
+            return;
+        }
+
+        mAssetRatioController =
```

Comment:
we want to do all mojo connections in the parent Activity, `BraveWalletActivity` in that case. And get all interfaces from there.

Reply:
thanks, fixed with https://github.com/brave/brave-core/pull/10174/commits/6cb21e371d38518475bf20df2a47d0fe7edcef5b